assets/js/src/core/components/accordion/accordion.tsx

```diff
@@ -11,24 +11,117 @@
 *  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
 */
 
-import React from 'react'
-import { Collapse, type CollapseProps } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { Collapse, type CollapseProps, Flex } from 'antd'
 import { useStyles } from '@Pimcore/components/accordion/accordion.styles'
+import { type ItemType } from 'rc-collapse/es/interface'
+import i18n from 'i18next'
+import { IconButton } from '@Pimcore/components/icon-button/icon-button'
+
+export interface AccordionItemType extends ItemType {
+  title: React.ReactElement
+  subtitle?: React.ReactElement
+}
 
 export interface AccordionProps extends CollapseProps {
-  exclusive?: boolean
+  items: AccordionItemType[]
+  spaced?: boolean
+  alignChevronLeft?: boolean
 }
 
-export const Accordion = ({ items, exclusive = true, ...props }: AccordionProps): React.JSX.Element => {
+export const Accordion = ({
+  items,
+  accordion = false,
+  spaced = false,
+  className,
+  activeKey,
+  alignChevronLeft = false,
```

Comment:
I would reuse or overwrite the api from ant here. 

https://ant.design/components/collapse#collapse
expandIconPosition => it can be start or end. 

Maybe we start with `start` and `after-title`  for now.
And when we have to add something new we can figure out how to do it.

The boolean value would limit us to only two positions.

Reply:
That makes sense. I will give that a go.

Reply:
I will use the same overwrite logic for the "collapse="icon"" property that is used in items atm to disable collapsing.

Reply:
Done :)